aspnetcore/fundamentals/startup.md

```diff
@@ -59,6 +59,34 @@ For features that require substantial setup, there are `Add[Service]` extension
 
 [!code-csharp[](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
+::: moniker range=">= aspnetcore-2.1"
+
+<a name="setcompatibilityversion"></a>
+### SetCompatibilityVersion
```

Comment:
Is this doc specific to MVC?

This is an MVC only feature. We might add this kind of thing in other places in the future if there's a good reason to.

I don't want people to get the impression that this will apply to all 2.1 changes in general. This is targeted at specific places where we changed MVC knowingly and with the expectation that it could break someone.